components/tikv_util/src/security.rs

```diff
@@ -24,6 +24,7 @@ pub struct SecurityConfig {
     #[serde(skip)]
     pub override_ssl_target: String,
     pub cert_allowed_cn: HashSet<String>,
+    pub redact_info_log: Option<bool>,
```

Comment:
What's the difference between `Some(true)`, `Some(false)` and `None`?

Reply:
If the option is not set by user, it defaults to false normally to keep existing behavior, but defaults to true if encryption is on.